### ISSUE TITLE
Significant discrepancy between reproduction results and those reported in the paper

### ISSUE THREAD

**Issue by rf-tang** [mrjohnmpu/UMPPI#4]
Dear author,

Thank you very much for sharing your protein–peptide dataset and the cold-start splitting strategies under various thresholds. I truly appreciate your effort in making the code and data publicly available.

I am currently trying to reproduce your model using the provided code and settings. Specifically, I used the new_protein_balanced split with a threshold of 0.5 and set model_mode=2. However, the results I obtained from 5-fold cross-validation are as follows:

AUC: 0.706, 0.502, 0.507, 0.502, 0.511

AP: 0.701, 0.501, 0.505, 0.502, 0.516

These results deviate significantly from the bar charts presented in your paper. I am concerned that I might have misunderstood or incorrectly reproduced the data splitting strategy or model configuration.

Could you kindly check whether the dataset splits and experimental settings used in your paper are consistent with those in the public repository? I would greatly appreciate any guidance on how to correctly reproduce the results.

Thank you very much for your time and help. I look forward to your reply.

Best regards

**Comment by rf-tang** [comment]
My config settings are as follows, and then I run "python main.py"

import argparse

def get_train_config():
	parse = argparse.ArgumentParser(description='train model')

	# preoject setting
	parse.add_argument('-setting', type=str, default='new_protein', help='setting')
	parse.add_argument('-clu-thre', type=str, default='0.5', help='clu_thre')
	parse.add_argument('-learn-name', type=str, default='binary_model', help='learn name')
	parse.add_argument('-save-best', type=bool, default=False, help='if save parameters of the current best model ')
	parse.add_argument('-threshold', type=float, default=0.80, help='save threshold')
	# model parameters
	parse.add_argument('-pad-pep-len', type=int, default=50, help='number of sense in multi-sense')
	parse.add_argument('-pad-prot-len', type=int, default=679, help='number of sense in multi-sense')
	# training parameters
	parse.add_argument('-model-mode', type=int, default=2, help='model mode')
	parse.add_argument('-if_multi_scaled', type=bool, default=False, help='if using k-mer ')
	parse.add_argument('-lr', type=float, default=1e-5, help='learning rate')
	parse.add_argument('-reg', type=float, default=1e-5, help='weight lambda of regularization')
	parse.add_argument('-accum-steps', type=int, default=1, help='Gradient Accumulation')
	parse.add_argument('-batch-size', type=int, default=128, help='number of samples in a batch')
	parse.add_argument('-epoch', type=int, default=25, help='number of iteration')
	parse.add_argument('-k-fold', type=int, default=5, help='k in cross validation,-1 represents train-test approach')
	parse.add_argument('-num-class', type=int, default=2, help='number of classes')
	parse.add_argument('-cuda', type=bool, default=True, help='if use cuda')
	parse.add_argument('-device', type=int, default=5, help='device id')
	parse.add_argument('-interval-log', type=int, default=16,
                       help='how many batches have gone through to record the training performance')
	parse.add_argument('-interval-valid', type=int, default=1,
                       help='how many epoches have gone through to record the validation performance')
	parse.add_argument('-interval-test', type=int, default=1,
                       help='how many epoches have gone through to record the test performance')
	config = parse.parse_args()
	return config
